bentoml/cli/__init__.py

```diff
@@ -12,405 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import re
```

Comment:
FYI: it looks like you moved this stuff to `bento_service.py`. You can use `git mv` or `git cp` when you migrate a lot of code from one file into a new file--that way, the new file will retain the old file's version history (and diffs will be more reasonable).